Makefile

```diff
@@ -42,6 +42,10 @@ nodelink-controller:
 machine-healthcheck:
 	$(DOCKER_CMD) ./hack/go-build.sh machine-healthcheck
 
+.PHONY: gen-crd
+gen-crd:
```

Comment:
I like it.  Perhaps we can add this script before as the first step in our make fmt, as that will detect any differences in git diff to ensure we don't have drift;  Anything that would change a CRD should generate before push.

Reply:
We can't do it right know since the generated CRDS for machine and machineset removes `additionalPrinterColumns`. And those are aws and openshift specific.

Reply:
@ingvagabund are you saying, after generating CRDs, someone needs to modify them?

Reply:
correct

Reply:
> correct

Oh - I would say this is bad. Somebody at some point will not realise and they things will go amiss.

Reply:
what about we drop the step for copying the files and only automate the generation

Reply:
Can't we just add the `kubebuilder:printcolumn` tags in the source so the `additionalPrinterColumns` is generated as well?

Reply:
Any example how to do it?

Reply:
I am pretty sure you mean https://github.com/openshift/machine-api-operator/pull/257/commits/0da2ff4ea36c70f002810b3d77f6a656595f1329